locked-asset/factory/src/locked_asset.rs

```diff
@@ -3,16 +3,25 @@ elrond_wasm::derive_imports!();
 
 use common_structs::*;
 
+use crate::attr_ex_helper;
+
 pub const ONE_MILLION: u64 = 1_000_000u64;
 pub const TEN_THOUSAND: u64 = 10_000u64;
 pub const PERCENTAGE_TOTAL: u64 = 100;
+pub const PERCENTAGE_TOTAL_EX: u64 = 100_000u64;
 pub const MAX_MILESTONES_IN_SCHEDULE: usize = 64;
 pub const DOUBLE_MAX_MILESTONES_IN_SCHEDULE: usize = 2 * MAX_MILESTONES_IN_SCHEDULE;
 
 #[derive(ManagedVecItem)]
 pub struct LockedToken<M: ManagedTypeApi> {
     pub token_amount: EsdtTokenPayment<M>,
-    pub attributes: LockedAssetTokenAttributes<M>,
+    pub attributes: LockedAssetTokenAttributesEx<M>,
+}
+
+#[derive(ManagedVecItem)]
+pub struct LockedTokenEx<M: ManagedTypeApi> {
```

Comment:
I do not see any difference between LockedToken and LockedTokenEx as a structure. What am I missing ?

Reply:
LockedAssetTokenAttributes```Ex```

Reply:
Both are with "Ex"

Reply:
ohhh, right, should have been without Ex, the old struct I mean. removed the struct entirely since only the Ex version will be used from now on